### PR TITLE
Cleanup in grammars

### PR DESCRIPTION
**Pull request by blitz-1306**
## Premise
For last few weeks Scribble received several bigger updates, that were affecting grammar. We figured out that it would be simplier to improve grammars to make overall style a bit more readable for a future changes.

## Changes
Tweaked `import_directive.pegjs` and `expr_grammar.pegjs`:
- [x] Tweaked indentation to consistently use 4 spaces instead of mixed `2` or `4`.
- [x] Reduced nesting in macros.
- [x] Conditional (ternary) operations in macros are now using same style `x === null ? undefined : x` instead of mixed cases.
- [x] Equality sign now always appear in same line as the rule name. Rule definition is staring on the next line (except `Keyword`s section).

Regards.

### CODE REVIEW

File: src/spec-lang/expr_grammar.pegjs [hunk 1]
```diff
@@ -279,233 +367,366 @@ SingleEscapeChar
...
+    }
+
+BitwiseORExpression =
+    head: BitwiseXORExpression
+    tail: (__ "|" __ BitwiseXORExpression)* {
+        return buildBinaryExpression(head, tail, location());
+    }
+
+RelationalExpression =
+    (
```
Comment (cd1m0) [inline src/spec-lang/expr_grammar.pegjs:512]: Same here - rule on one line, code on the next line(s).

**Review by cd1m0 — CHANGES_REQUESTED** [review]
Some nits

File: src/spec-lang/expr_grammar.pegjs [hunk 2]
```diff
@@ -1,126 +1,193 @@
...
-  type: FORALL __ "(" __ itr_type: IntType __ iterator: Identifier __ IN __ range: Range __ ")" __ expr: Expression
-  {
-    if(Array.isArray(range)) {
-      const [start, end] = range;
-      return new SForAll(itr_type, iterator, expr, start, end, undefined, location());
-    }
-    else {
-      return new SForAll(itr_type, iterator, expr, undefined, undefined, range, location());
+    type: FORALL
+    __ "("
```
[the remark this answers is not in the document]
Comment (blitz-1306) [reply src/spec-lang/expr_grammar.pegjs]: Addressed in https://github.com/ConsenSys/scribble/pull/49/commits/3554091ae185b1e77d31f3b8930caae285aefee5

File: src/spec-lang/expr_grammar.pegjs [hunk 3]
```diff
@@ -1,126 +1,193 @@
...
-  }
-
-UserFunctionDefinition = 
-  type: DEFINE __ label: AnnotationLabel? __ name: Identifier __ "(" __ args: TypedArgs? __ ")" __ returnType: Type __ "=" __ body: Expression
-  {
-    return new SUserFunctionDefinition(name, args === null ? [] : args, returnType, body, label !== null ? label : undefined, location());
-  }
-
-// Terminals
+    ("/" __)?
```
[the remark this answers is not in the document]
Comment (blitz-1306) [reply src/spec-lang/expr_grammar.pegjs]: Addressed in https://github.com/ConsenSys/scribble/pull/49/commits/3554091ae185b1e77d31f3b8930caae285aefee5

File: src/spec-lang/expr_grammar.pegjs [hunk 4]
```diff
@@ -1,126 +1,193 @@
...
+                return acc;
+            },
+            [head]
+        );
+    }
 
-WhiteSpace "whitespace"
-  = PrimitiveWhiteSpace
-  / LineTerminator PrimitiveWhiteSpace* ("*" / "///")
+UserFunctionDefinition =
```
[the remark this answers is not in the document]
Comment (blitz-1306) [reply src/spec-lang/expr_grammar.pegjs:120]: Addressed in https://github.com/ConsenSys/scribble/pull/49/commits/3554091ae185b1e77d31f3b8930caae285aefee5

**Review by cd1m0 — APPROVED** [review]
lgtm

**Pull request merged**